tests/testthat/test_system.R

```diff
@@ -272,6 +272,11 @@ test_that("prefecturecode", {
   res <- exploratory::prefecturecode(df$kanji.with.todofuken, output_type="name")
   expect_equal(FALSE, any(is.na(res)))
 
+  df <- data.frame(a=c("\u5317\u6D77\u9053", "\u6771\u4eac", "", NA, "\u304a\u304a\u3055\u304b"))
```

Comment:
Please add comment on what they are in Roma-ji, so that what is going on is understandable.

Reply:
Added comment.